src/views/product-tutorials-view/helpers/get-inline-content.ts

```diff
@@ -0,0 +1,44 @@
+import { getCollections } from 'lib/learn-client/api/collection'
+import { getTutorials } from 'lib/learn-client/api/tutorial'
+import {
+  Tutorial as ClientTutorial,
+  Collection as ClientCollection,
+} from 'lib/learn-client/types'
+
+export interface InlineTutorials {
+  [slug: string]: ClientTutorial[]
+}
+
+export interface InlineCollections {
+  [slug: string]: ClientCollection[]
+}
+
+export async function getInlineTutorials(
+  tutorialSlugs: string[]
+): Promise<InlineTutorials> {
+  const tutorials = (await getTutorials(
+    tutorialSlugs,
+    false
+  )) as ClientTutorial[]
+
+  const formattedTutorials = tutorials.reduce((acc, tutorial) => {
```

Comment:
I removed a bunch of[ formatting and data augmentation](https://github.com/hashicorp/learn/blob/231fd177b326529f4108dafbbd3b1e8ab1d53ecb/lib/data-fetching/utils/get-inline-content.ts) here to just return the plain client data shape. This may affect the blocks a bit, but I think this is a cleaner way to handle it at this level.